File: app/src/main/java/com/marchelo/developerslite/model/CommentsListHolder.java
package com.marchelo.developerslite.model;

import com.google.gson.annotations.Expose;

import java.util.ArrayList;
import java.util.List;

/**
 * @author Oleg Green
 * @since 16.05.16
 */
public class CommentsListHolder {
    @Expose
    private List<Comment> comments = new ArrayList<>();

    public List<Comment> getComments() {
        return comments;
    }

    @Override
    public String toString() {
        return "CommentsListHolder{" +
                "comments=" + comments +
                '}';
    }
}

File: app/src/main/java/com/marchelo/developerslite/model/CommentsListHolder.kt
package com.marchelo.developerslite.model

import com.google.gson.annotations.Expose

import java.util.ArrayList

/**
 * @author Oleg Green
 * *
 * @since 16.05.16
 */
class CommentsListHolder {
    @Expose
    val comments: List<Comment> = ArrayList()

    override fun toString(): String {
        return "CommentsListHolder{" +
                "comments=" + comments +
                '}'
    }
}
